src/server.py

```diff
@@ -72,4 +71,4 @@ async def run_subprocess():
 
 
 if __name__ == "__main__":
-    app.run(port=os.getenv("PORT"))
+    app.run(host="0.0.0.0", port=os.getenv("PORT"))
```

Comment:
@ryansurf 
Hey, here's a handy tip to keep in mind when working with Flask apps in Docker containers! If you want to access your Flask app from outside the container, make sure to set `host="0.0.0.0"`. 
This little configuration is crucial because it allows your Flask app to accept requests from all network interfaces, making it possible for your host machine to communicate with the container seamlessly. 
Don't forget this small but important detail, and you'll be good to go!

Reply:
Nice! Definitely a big oversight, I haven't been testing the Docker functionality much. Might be a good idea to implement some automated tests for it